### PR TITLE
Multiple sensor example

### PR DESCRIPTION
**Pull request by ba58smith**
Several people have asked on Slack about using more than one sensor in a single main.cpp. This should address that.

### CODE REVIEW

File: examples/multiple_sensors_example.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const char* analog_in_config_path = "/indoor_illuminance/analog_in";
+  const char* linear_config_path = "/indoor_illuminance/linear";
+
+  // Create a sensor that is the source of our data, that will be read every 500
+  // ms. It's a light sensor that's connected to the ESP's AnalogIn pin. The
+  // AnalogIn pin on ESP8266 is always A0, but ESP32 has many pins that can be
+  // used for AnalogIn, and they're expressed here as the XX in GPIOXX.
+  // When it's dark, the sensor's output (as read by analogRead()) is 120, and
+  // when it's bright, the output is 850, for a range of 730.
+  uint8_t pin = A0;
```
Comment (mairas) [inline examples/multiple_sensors_example.cpp]: To make the example build with ESP32 as is, could you have an ifdef to define the pin? Such as
```
#if defined(ESP8266)
#define ANALOG_INPUT_PIN A0
#elif defined(ESP32)
#define ANALOG_INPUT_PIN 12
#endif
```
after the includes?

File: examples/multiple_sensors_example.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  const float offset = -16.44;
+
+  // Wire up the output of the analog input to the Linear transform,
+  // and then output the results to the Signal K server. As part of
+  // that output, send some metadata to indicate that the "units"
+  // of the value we are sending is a "ratio" - which is the official
+  // unit type for a percentage represented as a float between 0.0 and 1.0
+  // See https://github.com/SignalK/specification/blob/master/schemas/definitions.json#L87
+  // for more details.
+  analog_input->connect_to(new Linear(multiplier, offset, linear_config_path))
```
Comment (mairas) [inline examples/multiple_sensors_example.cpp]: I don't actually see multiple sensors in this example?

**Review by mairas — COMMENTED** [review]
Sorry, I've been so buried with the SH-ESP32 work that I didn't notice this PR at all. The major comment is that I don't actually see multiple sensors in this example.

File: examples/multiple_sensors_example.cpp (hunk 2 above)
Comment (ba58smith) [reply examples/multiple_sensors_example.cpp]: You don't? There are three: temperature and humidity at the top, then the DigitalInutCounter at the bottom. You may have been thrown off by the IFDEF's in the middle, with two sensors above it. I can move those all to the top of the file if you like?

Comment (mairas) [reply examples/multiple_sensors_example.cpp]: Nevermind! I was a doofus and was only staring at one of the two commits! It's perfect as is.

**Comment by mairas** [comment]
As said, I was only staring at one of the two commits. It's great as is! Merging.

**Pull request merged**